src/containers/Map/Map.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import ReactMapGL, { FlyToInterpolator, ViewState } from "react-map-gl";
+import ZoomControl from "./ZoomControl";
+import "mapbox-gl/dist/mapbox-gl.css";
+import { easeCubicInOut } from "d3-ease";
+
+// const { token: accessToken } = require("config.json");
+type MapProps = {
+  mobile: boolean;
+  view: {
+    latitude: number;
+    longitude: number;
+    zoom: number;
+  };
+};
+
+const Map: React.FC<MapProps> = ({ mobile, view }) => {
+  const accessToken = process.env.REACT_APP_MAPBOX_TOKEN;
```

Comment:
I think this should stay outside the component, since the component may re-render, such that the token doesn't initialize again